system-test/system_test.go

```diff
@@ -81,9 +103,9 @@ VERSION=$(node -e "console.log(require('./package.json').version);")
 PROFILER="{{.PprofDir}}/pprof-$VERSION.tgz"
 
 # Create and set up directory for running benchmark.
-TESTDIR="$BASE_DIR/{{.Name}}"
+TESTDIR="{{.PprofDir}}/run-system-test/{{.Name}}"
```

Comment:
"pprof dir" is rather confusing - as if it's a directory where pprof is installed. Can it be named differently? Not "PprofNodejsDir" though.

Reply:
Done.